lib/StaticNarrative/StaticNarrativeImpl.py

```diff
@@ -56,9 +61,8 @@ def create_static_narrative(self, ctx, params):
         # ctx is the context object
         # return variables are: output
         # BEGIN create_static_narrative
-        snc = StaticNarrativeCreator(self.config)
+        snc = StaticNarrativeCreator(self.config, token=ctx["token"])
```

Comment:
alter the set up of the SNC class and the params that are passed into `create_static_narrative`